blockchain/chain_sync/src/sync.rs

```diff
@@ -267,17 +282,14 @@ where
                             .await?;
 
                         for b in ts_bundle.chain {
-                            let ts = ts[i as usize].clone();
                             // construct full tipsets from fetched messages
-                            let fts = self.tipset_msgs(
```

Comment:
Happy to see this method removed, however, I think we still need to include the BLOCK_MESSAGE_LIMIT check somewhere?

Reply:
I thought that was already validated for some reason, will add it in